File: feladatgyujtemeny/programozas/csharpmegoldas/feladat_014.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Feladatgyujtemeny
{
    class Program
    {
        static void Main(string[] args)
        {
            // Írj programot, ami beolvassa a körív sugarát és középponti szögét, és kiírja a körív
            // területét és a határoló ív hosszát!

            // Adatszerkezet (Data Structure)
            // (Globális) Változók - deklarálása (és ha lehet definiálása)
            int sugar = 0;
            int kozpontiSzog = 0;
            double pi = Math.PI;
            double ivHossz = 0.0d;
            double korivTerulet = 0.0d;
            string kimenet = "";

            // Felhasználói felület (User Interface) - bekérés
            Console.Write("Kérem a körív sugarát: ");
            sugar = Int32.Parse(Console.ReadLine());
            Console.Write("Kérem a körív középponti szögét: ");
            kozpontiSzog = Int32.Parse(Console.ReadLine());

            // Üzleti logika (Business Logic)
            ivHossz = (sugar * pi * kozpontiSzog) / 180;
            korivTerulet = (sugar * ivHossz) / 2;

            // Felhasználói felület (User Interface) - kiíratás
            kimenet = $"A körív sugara: {sugar} és központi szöge: {kozpontiSzog}.";
            kimenet += $"\nÍvhossza: {ivHossz}";
            kimenet += $"\nTerülete: {korivTerulet}";
            Console.WriteLine(kimenet);

            Console.ReadKey(true);
        }
    }
}

File: feladatgyujtemeny/programozas/pythonmegoldas/feladat_014.py
# Írj programot, ami beolvassa a körív sugarát és középponti szögét, és kiírja a körív
# területét és a határoló ív hosszát!

# Szükséges modulok importálása
import math

# Adatszerkezet (Data Structure)
# (Globális) Változók definiálása
sugar = 0
kozpontiSzog = 0
pi = math.pi
ivHossz = 0
terulet = 0
kimenet = ""

# Felhasználói felület (User Interface) - bekérés
sugar = int(input("Kérem a körív sugarát: "))
kozpontiSzog = int(input("Kérem a körív középponti szögét: "))

# Üzleti logika (Business Logic) 
ivHossz = (sugar * pi * kozpontiSzog) / 180
korivTerulet = (sugar * ivHossz) / 2

# Felhasználói felület (User Interface) - kiíratás
kimenet = f"A körív sugara: {sugar} és központi szöge: {kozpontiSzog}."
kimenet += f"\nÍvhossza: {ivHossz}"
kimenet += f"\nTerülete: {korivTerulet}"
print(kimenet)